<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MdViewerWidgetClass</class>
 <widget class="QWidget" name="MdViewerWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1057</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="Mantid::Vates::SimpleGui::TimeControlWidget" name="timeControlWidget" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Mantid::Vates::SimpleGui::ModeControlWidget" name="modeControlWidget" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="pqCameraToolbar" name="cameraWidget" native="true"/>
     </item>
     <item>
      <widget class="QToolButton" name="parallelProjButton">
       <property name="toolTip">
        <string>Toggle Parallel Projection</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons/ViewWidgetsIcons.qrc">
         <normaloff>:/VatesSimpleGuiViewWidgets/icons/parallel_projection.png</normaloff>:/VatesSimpleGuiViewWidgets/icons/parallel_projection.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="Mantid::Vates::SimpleGui::ColorSelectionWidget" name="colorSelectionWidget" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="pqAxesToolbar" name="axesWidget" native="true"/>
     </item>
     <item>
      <widget class="QToolButton" name="resetCenterToPointButton">
       <property name="toolTip">
        <string>Reset Center to Point</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons/ViewWidgetsIcons.qrc">
         <normaloff>:/VatesSimpleGuiViewWidgets/icons/pqResetCenterToPoint24.png</normaloff>:/VatesSimpleGuiViewWidgets/icons/pqResetCenterToPoint24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>3</number>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
<<<<<<< HEAD
      <property name="handleWidth">
       <number>3</number>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
=======
      <widget class="pqPipelineBrowserWidget" name="pipelineBrowser" native="true">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QTabWidget" name="propInfoTabs">
       <property name="currentIndex">
        <number>0</number>
>>>>>>> origin/master
       </property>
       <widget class="pqPipelineBrowserWidget" name="pipelineBrowser" native="true">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
       </widget>
<<<<<<< HEAD
       <widget class="QTabWidget" name="propInfoTabs">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="properties2">
         <attribute name="title">
          <string>Properties</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="pqProxiesWidget" name="proxiesPanel" native="true"/>
          </item>
        </layout>
        </widget>
        <widget class="QWidget" name="properties">
         <attribute name="title">
          <string>Properties</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="pqPropertiesPanel" name="propertiesPanel" native="true"/>
          </item>
         </layout>
        </widget>

        <widget class="QWidget" name="information">
         <attribute name="title">
          <string>Information</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
=======
       <widget class="QWidget" name="information">
        <attribute name="title">
         <string>Information</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>274</width>
              <height>370</height>
             </rect>
>>>>>>> origin/master
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>105</width>
               <height>387</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="pqProxyInformationWidget" name="proxyInfo" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="viewWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>600</height>
        </size>
       </property>
      </widget>
     </widget>
<<<<<<< HEAD
     <widget class="QDockWidget" name="colorMapEditorDock">
      <property name="windowTitle">
       <string>Color Map Editor</string>
=======
     <widget class="QWidget" name="viewWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
>>>>>>> origin/master
      </property>
      <widget class="pqColorMapEditor" name="colorMapEditorPanel">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="pqStatusBar" name="statusBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqPipelineBrowserWidget</class>
   <extends>QWidget</extends>
   <header>pqPipelineBrowserWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqStatusBar</class>
   <extends>QStatusBar</extends>
   <header>pqStatusBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqProxyInformationWidget</class>
   <extends>QWidget</extends>
   <header>pqProxyInformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqPropertiesPanel</class>
   <extends>QWidget</extends>
   <header>pqPropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Mantid::Vates::SimpleGui::ModeControlWidget</class>
   <extends>QWidget</extends>
   <header>MantidVatesSimpleGuiQtWidgets/ModeControlWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Mantid::Vates::SimpleGui::TimeControlWidget</class>
   <extends>QWidget</extends>
   <header>MantidVatesSimpleGuiViewWidgets/TimeControlWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqCameraToolbar</class>
   <extends>QWidget</extends>
   <header>pqCameraToolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Mantid::Vates::SimpleGui::ColorSelectionWidget</class>
   <extends>QWidget</extends>
   <header>MantidVatesSimpleGuiViewWidgets/ColorSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqAxesToolbar</class>
   <extends>QWidget</extends>
   <header>pqAxesToolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqColorMapEditor</class>
   <extends>QWidget</extends>
   <header>pqColorMapEditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqProxiesWidget</class>
   <extends>QWidget</extends>
   <header>pqproxieswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons/ViewWidgetsIcons.qrc"/>
 </resources>
 <connections/>
</ui>
